R/quantifyTranscriptionFactorSimilarity.R

```diff
@@ -0,0 +1,139 @@
+# quantifyTranscriptionFactorSimilarity.R
+#
+# Purpose: To quantify similarity between TF co-occurence and absence
+# Version: 1.0
+# Date: 2019-03-25
+# Author: Rachel Woo
+# License: MIT
+#
+# Input: An excel spreadsheet of the system parsed using the BCB420-2019-resources scripts.
+# Output: Dengrogram of TF  co-occurance written to
+#         ~/inst/img/TFOccuranceDendrogram.jpg
+# Dependencies: xlsx, tidyverse, readxl
+# ==============================================================================
+
+# SIDE EFFECTS:
+# Displays dendrogram
+
+# ==== FUNCTIONS ===============================================================
+
+#' getSysGenes
+#' Helper Function: Read system data from excel
+#' @author \href{https://orcid.org/ORCID:0000-0002-1387-487X}{Rachel Woo} (aut)
+#' @param filepath {string} to system excel sheet
+#' Helper function to read in system data from excel
+#' @return sysHGNC {list} A list of genes in the system
+#' @examples
+#' \donttest{
+#' getSysGenes("BCB420-2019-System-PHALY-0.3.xlsx")
+#' }
+getSysGenes <- function(filepath) {
```

Comment:
This is hopelessly outdated - and not at all compatible with the systems database. Too bad that you did not keep up communication about the changing landscape.

Reply:
Nb. There is a bit more information on the Student Wiki. At least I get the basic idea now: cluster genes based on presence-absence of transcription factors in the GTRD data set. I'm not sure that is biologically sound and would have been curious to see a few example of known co-regulated vs. not co-regulated genes. Also, this could probably be improved (if it works at all) by weighting TFs according to the number of sites.